website/docs/diff.md

```diff
@@ -27,6 +27,7 @@ slug: /diff/ # referenced in README.md and beacon
    - collection name must be valid UTF-8 characters;
 9. FerretDB offers the same validation rules for the `scale` parameter in both the `collStats` and `dbStats` commands.
    If an invalid `scale` value is provided in the `dbStats` command, the same error codes will be triggered as with the `collStats` command.
+10. FerretDB offers `debugError` command to return error for debugging.
```

Comment:
I'm not sure if that's something that should be here. The `debugError` is created mainly for our testing architecture to test how do we handle errors. @AlekSi wdyt?

Reply:
It should not be there. It also should not be in the FerretDB repo: https://github.com/FerretDB/FerretDB/issues/3672#issuecomment-1803276004